build.gradle

```diff
@@ -81,7 +81,7 @@ dependencies {
     implementation 'org.greenrobot:eventbus:3.2.0'
     implementation 'commons-cli:commons-cli:1.3.1'
     implementation 'org.bouncycastle:bcprov-jdk15on:1.67'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.2.1'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.6.1'
```

Comment:
Can jackson dependency be removed from security plugin given that it's been added in OpenSearch core?

Reply:
I've created https://github.com/opensearch-project/security/issues/1816 for follow up so we can avoid these kinds of problems in the future